### PR TITLE
Resource and item names

### PR DESCRIPTION
**Pull request by Chris-Baker**
- Add more resource names; Iron and Stone. 
- Add support for item names

image

### CODE REVIEW

File: src/types/queries.tsx [hunk 1]
```diff
@@ -675,9 +675,9 @@ export const StateFragmentFragmentDoc = gql`
             id
             coords: keys
             biome: value(match: { via: [{ rel: "Biome" }] })
-            bags: nodes(match: { kinds: ["Bag"], via: { rel: "Equip", dir: OUT } }) {
+            bags: nodes(match: { kinds: ["Bag", "Item"], via: { rel: "Equip", dir: OUT } }) {
```
Comment (chrisfarms) [inline src/types/queries.tsx]: is that one right? ... fetching equip'd Items under the "bags" section .... does _kinda_ make sense, but looks like a an accident

Comment (hypnoshock) [reply src/types/queries.tsx]: Looks like an accident, items aren't equipped directly to seekers (yet)
The other part of the query looked right
`slots: edges(match: {kinds: ["Resource", "Item"], via: {rel: "Balance"}}) {`

Comment (Chris-Baker) [reply src/types/queries.tsx:678]: We don't have an equiped section yet so they go into a bag and we treat it like a stack of 1 item. It's the only way we can see them currently

Comment (Chris-Baker) [reply src/types/queries.tsx]: That does look a bit off

Comment (Chris-Baker) [reply src/types/queries.tsx]: Fixed

File: src/helpers/inventory.ts [hunk 2]
```diff
@@ -38,3 +38,10 @@ export const isSamePos = (a: TileCoords, b: TileCoords) => {
 
     return true;
 };
+
+export const getItemName = (itemID: string): string => {
+    const last15Bytes = itemID.slice(-30, -16);
```
Comment (hypnoshock) [inline src/helpers/inventory.ts:43]: Try this as `itemID.slice(-30)` as -16 would limit the name to 7 chars

**Review by hypnoshock — APPROVED** [review]
Approved but that getItemName function should have that -16 in the slice. I'm not sure if supplying a hex with null bytes to `toUtf8String` would be alright or not so would need testing. I reckon it'll be alright to do `itemID.slice(-30)`

**Pull request merged**